webdriver/tests/take_element_screenshot/user_prompts.py

```diff
@@ -15,56 +15,94 @@ def take_element_screenshot(session, element_id):
         )
     )
 
+@pytest.fixture
+def check_user_prompt_closed_without_exception(session, create_dialog):
+    def check_user_prompt_closed_without_exception(dialog_type):
+        session.url = inline("<input/>")
+        element = session.find.css("input", all=False)
+
+        create_dialog(dialog_type, text=dialog_type)
+
+        response = take_element_screenshot(session, element.id)
+        value = assert_success(response)
+
```

Comment:
I miss a call to `assert_dialog_handled()` here. Is there a reason why you aren't checking that?

Reply:
Added back assert_dialog_handled everywhere.